plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageInitialization.java

```diff
@@ -349,7 +350,11 @@ public void saveSettings(DBPDataSourceContainer dataSource) {
         bootstrap.setInitQueries(bootstrapQueries);
 
         confConfig.setKeepAliveInterval(keepAliveInterval.getSelection());
```

Comment:
You mustn't set connection settings from connection type.
DBPConnectionType must be used in runtime (as any other DBPConnectionType  properties)

Reply:
Get it, m I right that require new logic for close idle connections in class DataSourceMonitorJob ?